pkg/daemon/config.go

```diff
@@ -157,3 +157,37 @@ func (conf *ptp4lConf) renderPtp4lConf() (string, string) {
 	}
 	return configOut, strings.Join(conf.mapping, ",")
 }
+
+func addFlagsForMonitor(nodeProfile *ptpv1.PtpProfile, conf *ptp4lConf, stdoutToSocket bool) {
```

Comment:
Only required change in this PR looks like 
_, exist := conf.sections["[global]"].options["summary_interval"]
Why changing so many code?